### PR TITLE
Feature/sse s3

### PR DESCRIPTION
**Pull request by robparrott**
Adds a new `--use-sse` option that makes all uploads set with AES-256 server encryption.

Implements issue #106

### CODE REVIEW

File: internal/goofys.go [hunk 1]
```diff
@@ -462,6 +462,11 @@ func (fs *Goofys) copyObjectMultipart(size int64, from string, to string, mpuId
 			Key:          fs.key(to),
 			StorageClass: &fs.flags.StorageClass,
 			ContentType:  fs.getMimeType(to),
+			ServerSideEncryption:  &fs.flags.SSEType,
+		}
+
+		if  fs.flags.UseSSE  {
```
Comment (kahing) [inline internal/goofys.go]: Not at a computer now but this seems redundant?

**Comment by kahing** [comment]
Could you squash this?

File: internal/goofys.go (hunk 1 above)
Comment (robparrott) [inline internal/goofys.go]: Yup. My mistake. Good eye!

**Comment by robparrott** [comment]
Changed.

**Comment by kahing** [comment]
55c235cf0dec33bd5c0542e5c0ed4fe2ced7628f

**Pull request closed without merging**